test/distributed/test_distributed.py

```diff
@@ -1574,15 +1574,16 @@ def _test_all_to_all_helper(self, group, group_id, rank):
         self._barrier()
 
     @unittest.skipIf(
-        BACKEND != "mpi", "Only MPI supports CPU all_to_all_single"
+        BACKEND != "mpi" and BACKEND != "gloo",
+        "Only MPI and Gloo support CPU all_to_all_single"
     )
     def test_all_to_all_single_equal_split(self):
         group, group_id, rank = self._init_global_test()
         self._test_all_to_all_single_equal_split_helper(group, group_id, rank)
 
-    @unittest.skip("NCCL A2A is not enabled for OSS builds")
+    @unittest.skipIf(BACKEND == "nccl", "NCCL A2A is not enabled for OSS builds")
```

Comment:
Is this line redundant as the following line already skipped all non-gloo cases? Or is this just provides more info?

Reply:
Yes Shen. Skipping NCCL A2A was a workaround. Once we enable NCCL A2A, we can remove this line.